src/addcall.c

```diff
@@ -93,8 +93,7 @@ int addcall(void) {
     j = getctydata(hiscall);
     worked[i].country = j;
     if (strlen(comment) >= 1) {		/* remember last exchange */
-	strncpy(worked[i].exchange, comment, 12);
-	worked[i].exchange[12] = 0;
+	g_strlcpy(worked[i].exchange, comment, 12);
```

Comment:
The last parameter should be 13 or better sizeof(worked[i].exchange). 

g_strlcpy needs the size of the buffer, not the number bytes you want to copy.

Reply:
TNX, fixed.

I always forget that glib functions can be used here :)